packages/scripts/src/admin-proposals/addContractCreator.js

```diff
@@ -52,146 +35,143 @@ const argv = require("minimist")(process.argv.slice(), {
 });
 
 async function run() {
-  const { ethereum, polygon, verify } = argv;
-  const { web3, netId } = await _setupWeb3();
-
-  // Contract ABI's
-  const Registry = getContract("Registry");
-  const GovernorRootTunnel = getContract("GovernorRootTunnel");
-  const GovernorChildTunnel = getContract("GovernorChildTunnel");
-  const Governor = getContract("Governor");
-  const Finder = getContract("Finder");
-  const Voting = getContract("Voting");
+  const { ethereum, polygon, arbitrum, verify } = argv;
+  if (!(polygon || ethereum || arbitrum)) throw new Error("Must specify either --ethereum, --polygon or --arbitrum");
 
   // Parse comma-delimited CLI params into arrays
-  let ethereumContractToRegister = ethereum;
-  let polygonContractToRegister = polygon;
-  let crossChainWeb3;
-
-  // If polygon address is specified, initialize Governance relay infrastructure contracts
-  let polygon_netId;
-  let polygon_registry;
-  let polygon_governor;
-  if (!(polygonContractToRegister || ethereumContractToRegister))
-    throw new Error("Must specify either --ethereum or --polygon or both");
-  else if (polygonContractToRegister) {
-    if (!process.env.POLYGON_NODE_URL)
-      throw new Error("If --polygon is defined, you must set a POLYGON_NODE_URL environment variable");
-    crossChainWeb3 = new Web3(process.env.POLYGON_NODE_URL);
-    polygon_netId = await crossChainWeb3.eth.net.getId();
-    polygon_registry = new crossChainWeb3.eth.Contract(
-      Registry.abi,
-      await _getContractAddressByName("Registry", polygon_netId)
-    );
-    polygon_governor = new crossChainWeb3.eth.Contract(
-      GovernorChildTunnel.abi,
-      await _getContractAddressByName("GovernorChildTunnel", polygon_netId)
-    );
-  }
+  const networksToAdministrate = [];
+  if (polygon) networksToAdministrate.push(137);
+  if (arbitrum) networksToAdministrate.push(42161);
```

Comment:
so we'll need to add optimism and boba here as well?

Reply:
yep